### PR TITLE
add blank target to hosted url

### PR DESCRIPTION
**Pull request by chrispetito**
# Description

Describe change or new feature here.

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/components/SubmissionForm/ProjectDetails.js [hunk 1]
```diff
@@ -98,66 +129,73 @@ const ProjectDetails = props => {
...
+          variant="outlined"
           type="text"
           value={name} /*???*/
           required
           name="name"
           id="standard-required"
-          placeholder="App Name*"
-          margin="normal"
+          label="App Name"
+          // margin="normal"
```
Comment (lappjeff) [inline src/components/SubmissionForm/ProjectDetails.js]: Is this commented out `margin="normal"` doing anything?

File: src/styles/projectgallery.scss [hunk 2]
```diff
@@ -29,9 +29,8 @@
 .MuiFormControl-root {
       width: 360px;
       .MuiInputBase-input{
-            height:2.8em;
-            border: 1px solid rgba(128, 128, 128, 0.521);
-            padding: 5px;
+            height:100%;
+            // padding: 5px;
```
Comment (lappjeff) [inline src/styles/projectgallery.scss]: Padding line doing anything?

Comment (chrispetito) [reply src/styles/projectgallery.scss]: Removed!

File: src/components/SubmissionForm/ProjectDetails.js (hunk 1 above)
Comment (chrispetito) [reply src/components/SubmissionForm/ProjectDetails.js]: Wasn't doing anything with fixes I made. Removed.

**Pull request merged**